### PR TITLE
Add editor support entry for sublime text

### PR DESCRIPTION
**Pull request by toreriklinnerud**
As requested in https://github.com/penelopezone/rubyfmt/issues/126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,21 @@ Once installed, add the following to vscode's `settings.json` file:
   },
 ```
 
+### Sublime Text
+
+Install the [rubyfmt plugin](https://github.com/toreriklinnerud/sublime-rubyfmt/) from [Package Control](https://packagecontrol.io): Install Package -> rubyfmt. 
+
+`ruby` and `rubyfmt` are assumed to be on path. Ruby files are formatted on save or by pressing `Alt + ;` or on macOS: `Cmd + ;`
```
Comment (fables-tales) [inline README.md]: why does ruby need to be on the path?

**Comment by fables-tales** [comment]
could you rebase? Sorry we're getting lots of readme updates right now

File: README.md (hunk 1 above)
Comment (toreriklinnerud) [reply README.md]: I had the problem where rubyfmt would crash when there were syntax errors in the ruby file so if rubyfmt exits with an error I'm invoking `ruby -c` to check for syntax errors and surface them in the editor window to give the user an idea of why the file isn't being formatted.

**Comment by toreriklinnerud** [comment]
Rebased

**Pull request merged**